helper.psm1

```diff
@@ -189,15 +202,13 @@ Function showConfig
     Write-Host "Workdir        : "$WORKDIR
     Write-Host "Workspace      : "$env:WORKSPACE
     Write-Host "------------------------------------------------------------------------------"
+    Write-Host "Clcache Statistics"
+    configureCache
```

Comment:
showConfig should not call configureCache, it should only show things

Reply:
added new commit to fix issue